packages/airnode-protocol/contracts/rrp/requesters/RrpBeaconServer.sol

```diff
@@ -245,6 +245,7 @@ contract RrpBeaconServer is
         bytes32 beaconId = requestIdToBeaconId[requestId];
         require(beaconId != bytes32(0), "No such request made");
         delete requestIdToBeaconId[requestId];
+        require(data.length == 64, "Incorrect data length");
```

Comment:
Is this from another branch?

Reply:
Yeah, I think github differ is broken - sorry I didn't notice that there are so many changes not from my PR... 
I'd have rebased it.